From 2f8a74523dd53f6d928700f1c0854563f2cc4a65 Mon Sep 17 00:00:00 2001
From: Mykola Lysenko <Mykola.Lysenko@amd.com>
Date: Fri, 11 Dec 2015 20:18:54 +0800
Subject: [PATCH 0947/1050] drm/amd/dal: remove SW payload row on reset

dc_helpers_dp_mst_write_payload_allocation_table required
passed stream connector to have crtc. However during reset
mode it does not have one as state got already updated.
Also it is not used

Change-Id: I9b0a3bd9bddf3c6754bbf724fd6d9dfb4f71dae5
Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Acked-by: Harry Wentland<harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dc_helpers.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dc_helpers.c b/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dc_helpers.c
index 87db164..f2d1dc8 100644
--- a/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dc_helpers.c
+++ b/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dc_helpers.c
@@ -187,10 +187,6 @@ bool dc_helpers_dp_mst_write_payload_allocation_table(
 	bool find_stream_for_sink;
 
 	aconnector = get_connector_for_sink(dev, stream->sink);
-	crtc = aconnector->base.state->crtc;
-
-	if (!crtc)
-		return false;
 
 	if (!aconnector->mst_port)
 		return false;
-- 
1.9.1

